python/langsmith/client.py

```diff
@@ -931,6 +932,43 @@ def request_with_retries(
                     filler = "*" * (max(0, len(api_key) - 7))
                     masked_api_key = f"{prefix}{filler}{suffix}"
 
+                    debug_crash_dump_file = ls_utils.get_env_var(
```

Comment:
can we read env var once in global scope and then just use that here (reading env vars is expensive)